### PR TITLE
Feat/sentry

### PR DESCRIPTION
**Pull request by tgmarinho**
Task: https://github.com/sibelius/ccsseraphini/issues/345

Done:
- [x] Add Sentry setup
- [x] Update README on **web** package with basic instructions
- [x] @sibelius should create a sentry project, or we can use the project that I have already set up in my account 
- [x] Should add the new .env in the Vercel, see the .env.sample

Missing:
- [ ] setup GitHub + sentry, to create issues automatically. But it's not related to the code, I put the links in the task.

Details:

Screen Shot 2022-04-07 at 22 25 42

Screen Shot 2022-04-07 at 22 26 00

### CODE REVIEW

**Review by NicolasLopes7 — APPROVED** [review]
LGTM!
image

File: apps/web/.env.example [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+
+## Discord and Discord CronJob - display notifications on discord channel #tweets
 WEBHOOK_DISCORD= #get this url creating a channel then create a webhook, copy the url and paste here
 CRONJOB_HEADER_KEY=# add a key here, should be the same as in cronjob.org, sample: x-servicename-supersecret-key
 CRONJOB_HEADER_VALUE=# add a value here, should be same as in cronjob.org , sample: 1q2w3e4r5t6y7u8i9o0p
+
+## Sentry - logging and debugging in production
+SENTRY_DSN=
+NEXT_PUBLIC_SENTRY_DSN=
+SENTRY_ORG=
```
Comment (sibelius) [inline apps/web/.env.example:19]: are we using this?

File: apps/web/.env.example [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+SENTRY_PROJECT=
+SENTRY_AUTH_TOKEN=
```
Comment (sibelius) [inline apps/web/.env.example:21]: same

File: apps/web/.env.example [hunk 3]
```diff
@@ -1,9 +1,21 @@
...
+## Discord and Discord CronJob - display notifications on discord channel #tweets
 WEBHOOK_DISCORD= #get this url creating a channel then create a webhook, copy the url and paste here
 CRONJOB_HEADER_KEY=# add a key here, should be the same as in cronjob.org, sample: x-servicename-supersecret-key
 CRONJOB_HEADER_VALUE=# add a value here, should be same as in cronjob.org , sample: 1q2w3e4r5t6y7u8i9o0p
+
+## Sentry - logging and debugging in production
+SENTRY_DSN=
+NEXT_PUBLIC_SENTRY_DSN=
+SENTRY_ORG=
+SENTRY_PROJECT=
```
Comment (sibelius) [inline apps/web/.env.example:20]: same

File: apps/web/.env.example (hunk 1 above)
Comment (tgmarinho) [reply apps/web/.env.example:19]: only sentry_dsn, removing the others here

File: apps/web/.env.example (hunk 2 above)
Comment (tgmarinho) [reply apps/web/.env.example:21]: https://github.com/sibelius/ccsseraphini/pull/414/files

File: apps/web/.env.example (hunk 3 above)
Comment (tgmarinho) [reply apps/web/.env.example:20]: https://github.com/sibelius/ccsseraphini/pull/414/files

**Comment by tgmarinho** [comment]
I removed some .env unnecessary 
see https://github.com/sibelius/ccsseraphini/pull/414/files

**Pull request merged**